docs/installation.rst

```diff
@@ -19,11 +19,12 @@ use the package manager `pip`_ to install the dependencies for FlaskBB.
 
 Virtualenv Setup
 ~~~~~~~~~~~~~~~~
-
-The easiest way to install `virtualenv`_ and
+**Linux:** The easiest way to install `virtualenv`_ and
 `virtualenvwrapper`_ is, to use the package manager on your system (if you
 are running Linux) to install them.
 
+**Windows:** Take a look at the `flask`_ documentation (then skip ahead to dependencies).
```

Comment:
I'm a little iffy about sending people to Flask's documentation for setting up a venv in windows, but since I don't have practical experience on windows, I guess deferring there for now is good.